app/assets/stylesheets/dashboard.css.scss

```diff
@@ -1,10 +1,78 @@
+$circle-color: rgba($primary-color,0.5);
+
+.dash-header {
+  background-color: $topbar-bg-color;
+}
+
+.avatar-preview img {
+  padding: 1em;
+}
+
 .circle-thumb {
-  background-color: rgba(125,0,255,0.5);
-  border-radius: 10px;
+  background-color: $circle-color;
+  border-radius: 7px;
   box-sizing: border-box;
-  padding: 1em;
+  height: 200px;
+  overflow: hidden;
+  position: relative;
+
+  .circle-name {
+    width: 100%;
+    position: absolute;
+    bottom: 0px;
+  }
+
+  .circle-pic img {
+    width: 100%;
+    @extend .hvr-grow;
+  }
+
+  div.circle-pic {
+    position: absolute;
+  }
+
+  div.circle-texts {
+    width: 100%;
+    height: 100%;
+    padding: 1em;
+    border-radius: 7px;
+    position: absolute;
+    color: #fff;
+    background-color: rgba(0, 0, 0, 0.45);
+    opacity: 0;
+    -webkit-transition: opacity .2s ease;
+    -moz-transition: opacity .2s ease;
+
+    span {
+      width:100%;
```

Comment:
Properties should be ordered height, left, position, top, width, z-index<br>Colon after property should be followed by one space